server/lib/config/pbench-put-shim.cfg

```diff
@@ -0,0 +1,4 @@
+[put-server]
+pbench_results_redirector = pbench.results.example.com
+pbench_web_server = pbench.example.com
+token = Authentication-token
```

Comment:
I have no problem with the existence of this file -- if you add
```
[config]
path = %(install-dir)s/lib/config
files = pbench-server-default.cfg
```
to it, then it could _be_ the shim's config file -- but, I don't think we need or want it committed to the repo.

Reply:
Sure

Reply:
just for reference, I have left it there for now. Will remove it before merge